pkgs/development/python-modules/shapely/default.nix

```diff
@@ -22,11 +22,16 @@ buildPythonPackage rec {
     export LANG="en_US.UTF-8";
   '';
 
-  patchPhase = let
+  postPatch =  let
     libc = if stdenv.isDarwin then "libc.dylib" else "libc.so.6";
```

Comment:
This could actually just be `"libc${stdenv.hostPlatform.extensions.sharedLibrary}"`.